google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -105,7 +105,10 @@
 
   <application-components>
     <component>
-      <implementation-class>com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent</implementation-class>
```

Comment:
I see this was removed here in plugin.xml but the class was not deleted/renamed. Also, where is all the logic that was this class now?